hyperspy/misc/io/fei_stream_readers.py

```diff
@@ -1,25 +1,20 @@
 import numpy as np
 import dask.array as da
+import sparse
 
 from hyperspy.decorators import jit_ifnumba
 
 
-try:
-    import sparse
```

Comment:
so `sparce` is going to be hard dependency ?

Reply:
It is already the case: https://github.com/hyperspy/hyperspy/blob/24619445e501cbc541d1b4179db31337d4959a8f/setup.py#L49-L67